src/data/api.js

```diff
@@ -1,16 +1,26 @@
 import { getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { logError } from '@edx/frontend-platform/logging';
 import { ExamAction } from '../constants';
 import { generateHumanizedTime } from '../helpers';
 
 const BASE_API_URL = '/api/edx_proctoring/v1/proctored_exam/attempt';
 
 async function fetchActiveAttempt() {
+  // fetch 'active' (timer is running) attempt if it exists
   const activeAttemptUrl = new URL(`${getConfig().EXAMS_BASE_URL}/api/v1/exams/attempt/latest`);
   const activeAttemptResponse = await getAuthenticatedHttpClient().get(activeAttemptUrl.href);
   return activeAttemptResponse.data;
 }
 
+async function fetchLatestExamAttempt(sequenceId) {
```

Comment:
basically the same thing as the above function. I broke this out to make reading the behavior a bit easier

Reply:
Would it be worth adding a comment noting this so that people don't confused by these functions having similar functionality later on?

Reply:
good idea 👍